instrumentation/pg/opentelemetry-instrumentation-pg.gemspec

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Copyright The OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'opentelemetry/instrumentation/pg/version'
+
+Gem::Specification.new do |spec|
+  spec.name        = 'opentelemetry-instrumentation-pg'
+  spec.version     = OpenTelemetry::Instrumentation::PG::VERSION
+  spec.authors     = ['OpenTelemetry Authors']
+  spec.email       = ['cncf-opentelemetry-contributors@lists.cncf.io']
+
+  spec.summary     = 'PG (PostgreSQL) instrumentation for the OpenTelemetry framework'
+  spec.description = 'PG (PostgreSQL) instrumentation for the OpenTelemetry framework'
+  spec.homepage    = 'https://github.com/open-telemetry/opentelemetry-ruby'
+  spec.license     = 'Apache-2.0'
+
+  spec.files = ::Dir.glob('lib/**/*.rb') +
+               ::Dir.glob('*.md') +
+               ['LICENSE', '.yardopts']
+  spec.require_paths = ['lib']
+  spec.required_ruby_version = '>= 2.5.0'
+
+  spec.add_dependency 'lru_redux', '~> 1.1'
```

Comment:
I picked this particular implementation of an LRU cache primarily because it seems to be far and away the most downloaded on rubygems. It's a poor proxy of "is this a good choice," but I feel good that many others have made the same decision. 😆